src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy

```diff
@@ -394,4 +395,30 @@ class IqPolicyEvaluatorTest
       1 * log.println('\nThe detailed report can be viewed online at http://server/report\n' +
           'Summary of policy violations: 0 critical, 0 severe, 0 moderate')
   }
+
+  def 'Resource not found exception handling'() {
```

Comment:
I would change the name of the test to reflect the more specific behavior of the code.

Reply:
> I would change the name of the test to reflect the more specific behavior of the code.

Changed 2be7c0a